ethereum/eth/src/main/java/tech/pegasys/pantheon/ethereum/eth/manager/EthScheduler.java

```diff
@@ -92,7 +101,11 @@ protected EthScheduler(
   }
 
   public Future<?> scheduleWorkerTask(final Runnable command) {
```

Comment:
`scheduleSyncWorkerTask`?

Reply:
Sure.  Also scheduleTxWorkerTask, so both align with the field name.